app/controllers/members/dues_controller.rb

```diff
@@ -9,6 +9,20 @@ def show
     end
   end
 
+  def cancel
+    message = "You don't have an active subscription"
```

Comment:
This message could be a little more specific. Maybe "You don't have an active membership dues subscription"